package.json

```diff
@@ -37,6 +38,7 @@
     "eslint": "1.3.1",
     "eslint-config-defaults": "6.0.0",
     "eslint-plugin-filenames": "0.1.2",
-    "eslint-plugin-react": "3.3.1"
+    "eslint-plugin-react": "3.3.1",
+    "mocha": "^2.3.2"
```

Comment:
In keeping with the fixed ("pinned") dependencies, let's do `2.3.2`, not `^2.3.2`--sure is a pain to remember, though. :(